src/main/java/frc/trigon/robot/subsystems/arm/toohardarm/ToohardArmConstants.java

```diff
@@ -0,0 +1,132 @@
+            ELEVATOR_ENCODER_ID = 1;
+    private static final boolean
+            MASTER_ANGLE_MOTOR_INVERTED = false,
+            FOLLOWER_ANGLE_MOTOR_INVERTED = false,
+            MASTER_ELEVATOR_MOTOR_INVERTED = false,
+            FOLLOWER_ELEVATOR_MOTOR_INVERTED = false;
+    private static final CANSparkMax.IdleMode
+            ANGLE_MOTORS_IDLE_MODE = CANSparkMax.IdleMode.kBrake,
+            ELEVATOR_MOTORS_IDLE_MODE = CANSparkMax.IdleMode.kBrake;
+    private static final int VOLTAGE_COMPENSATION_SATURATION = 12;
+    static final CANSparkMax
+            MASTER_ANGLE_MOTOR = new CANSparkMax(MASTER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ANGLE_MOTOR = new CANSparkMax(FOLLOWER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            MASTER_ELEVATOR_MOTOR = new CANSparkMax(MASTER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ELEVATOR_MOTOR = new CANSparkMax(FOLLOWER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+    private static final double
+            ANGLE_MOTOR_OFFSET = 0,
+            ELEVATOR_MOTOR_OFFSET = 0;
+    private static final SensorDirectionValue ANGLE_MOTOR_DIRECTION = SensorDirectionValue.Clockwise_Positive;
+    private static final boolean ELEVATOR_MOTOR_PHASE = false;
+    private static final AbsoluteSensorRangeValue ANGLE_MOTOR_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    public static final CANcoder ANGLE_ENCODER = new CANcoder(ANGLE_ENCODER_ID);
+    public static final TalonSRX ELEVATOR_ENCODER = new TalonSRX(ELEVATOR_ENCODER_ID);
+
+    private static final double
+            ANGLE_P = 0,
+            ANGLE_I = 0,
+            ANGLE_D = 0,
+            ELEVATOR_P = 0,
+            ELEVATOR_I = 0,
+            ELEVATOR_D = 0;
+    static final PIDController
+            ANGLE_PID_CONTROLLER = new PIDController(ANGLE_P, ANGLE_I, ANGLE_D),
+            ELEVATOR_PID_CONTROLLER = new PIDController(ELEVATOR_P, ELEVATOR_I, ELEVATOR_D);
+
+    private static final double
+            ANGLE_MOTOR_KS = 1,
+            ANGLE_MOTOR_KG = 1,
+            ANGLE_MOTOR_KV = 1,
+            ANGLE_MOTOR_KA = 1,
+            ELEVATOR_MOTOR_KS = 1,
+            ELEVATOR_MOTOR_KG = 1,
+            ELEVATOR_MOTOR_KV = 1,
+            ELEVATOR_MOTOR_KA = 1;
+    static final ArmFeedforward ANGLE_FEEDFORWARD = new ArmFeedforward(
+            ANGLE_MOTOR_KS,
+            ANGLE_MOTOR_KG,
+            ANGLE_MOTOR_KV,
+            ANGLE_MOTOR_KA
+    );
+    static final ElevatorFeedforward ELEVATOR_FEEDFORWARD = new ElevatorFeedforward(
+            ELEVATOR_MOTOR_KS,
+            ELEVATOR_MOTOR_KG,
+            ELEVATOR_MOTOR_KV,
+            ELEVATOR_MOTOR_KA
+    );
+
+    public static final StatusSignal<Double> ANGLE_ENCODER_POSITION_SIGNAL = ANGLE_ENCODER.getPosition();
+    public static final StatusSignal<Double> ANGLE_ENCODER_VELOCITY_SIGNAL = ANGLE_ENCODER.getVelocity();
```

Comment:
By definition, these should only be used in the ToohardArmIO, so these should not be public 
Also, there is no need for the redundant `public static final StatusSignal<Double>`